.github/workflows/conda-build.yml

```diff
@@ -10,8 +10,11 @@ on:
 
```

Comment:
looks like we need to replace `RELEASE_VERSION=$(date +%Y%m%d%H%M%S)` by something that runs on `windows-latest` runner

Reply:
good catch - also - it looks like the versioning scheme has recently been changed to calver, but no release has been done yet. I would strongly argue against date-based versioning. Meaningful pinning of this package for people that use it as a library is basically impossible with calver. Also it does not allow you to reason about versions in any meaningful way.

Of course this is also not automatically given with semver, and I have been fooled by relying on it as well. But most of the time I find it more useful than not. Only for end-user softwares (ubuntu) I can see a benefit in using calver. So... While orthogonal to this, would you be willing to reconsider?

Reply:
completely agree! It should be semver everywhere